### PR TITLE
100% code coverage milestone

### PR DESCRIPTION
**Pull request by paulmelnikow**

### CODE REVIEW

**Comment by paulmelnikow** [comment]
Did this file end up in the wrong folder?

**Comment by gr2m** [comment]
> Did this file end up in the wrong folder?

Yes, looks like it, can you rename it?

File: 100.tweet [hunk 1]
```diff
@@ -0,0 +1 @@
+Tests have been written, dead code has been stripped out, and the Nock codebase now has 100% code coverage! Thanks to all the contributors who worked on this! Special mention to @paulmelnikow who did a major lift on this work and Matt Wilson who finished the home stretch. Onward!
```
Comment (gr2m) [inline 100.tweet]: Maybe prefix with `📍 ` as a means to say we hit a milestone?

**Comment by gr2m** [comment]
Oops okay now the tweet is too long. You can remove the emoji prefix, or maybe reword the tweet a little to shorten it, e.g. 

```diff
- Tests have been written, dead code has been stripped out, and the Nock codebase ...
+ Tests have been written, dead code removed, and the Nock codebase...
```

**Review by gr2m — APPROVED** [review]
🐦👍

**Comment by gr2m** [comment]
Feel free to tweet anytime you like. I think now is a good time in terms of reaching as many nock users as possible
